cloudify-dns-bind9-openstack-blueprint.yaml

```diff
@@ -1,10 +1,10 @@
-tosca_definitions_version: cloudify_dsl_1_2
+tosca_definitions_version: cloudify_dsl_1_3
 
 imports:
 
-  - http://www.getcloudify.org/spec/cloudify/3.3.1/types.yaml 
-  - http://www.getcloudify.org/spec/openstack-plugin/1.3.1/plugin.yaml 
-  - http://www.getcloudify.org/spec/diamond-plugin/1.3.1/plugin.yaml 
+  - http://www.getcloudify.org/spec/cloudify/3.4rc1/types.yaml
```

Comment:
I'm testing using rc1. I'd need to test it on 3.4m5 to be sure everything's working fine. Is this small delay acceptable?